.github/workflows/raise-pr-for-constants.yml

```
@@ -0,0 +1,72 @@
+          DEST_REPO="rudderlabs/rudder-sdk-js"
+          git clone https://github.com/${DEST_REPO}.git dest-repo
+          cd dest-repo
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "noreply@github.com"
+          git config --global url."https://${{ secrets.GITHUB_TOKEN }}@github.com/".insteadOf "https://github.com/"
+          git checkout -b ${{ env.BRANCH_NAME }}
+          git pull origin ${{ env.BRANCH_NAME }} || echo "No updates to pull"
+          
+      - name: Compare Destinations.ts
+        id: compare_files
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2034:warning:2:1: DEST_REPO appears unused. Verify use (or export if used externally)

(shellcheck)